Rule34.xxx Downloader/Services/DownloadService.cs

```diff
@@ -29,6 +29,8 @@ public static void Download(string url, string filePath)
                 {
                     try
                     {
+                        client.DefaultRequestHeaders.Referrer = new Uri("https://rule34.xxx/");
```

Comment:
I believe this should be `System.Uri`, build fails in current state.
Other than that it works perfectly now
```suggestion
                        client.DefaultRequestHeaders.Referrer = new System.Uri("https://rule34.xxx/");
```